controller/collaborators.go

```diff
@@ -84,22 +85,29 @@ func (c *CollaboratorsController) List(ctx *app.ListCollaboratorsContext) error
 				}, "unable to find the identity listed in the space policy")
 				return errors.New("Identity listed in the space policy not found")
 			}
-			appUser := ConvertToAppUser(ctx.RequestData, &identities[0].User, &identities[0])
-			data[i] = appUser.Data
+			resultIdentities[i] = identities[0]
+			resultUsers[i] = identities[0].User
 			return nil
 		})
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal(err.Error()))
 		}
 	}
 
-	response := app.UserList{
-		Links: &app.PagingLinks{},
-		Meta:  &app.UserListMeta{TotalCount: count},
-		Data:  data,
-	}
-	setPagingLinks(response.Links, buildAbsoluteURL(ctx.RequestData), len(page), offset, limit, count)
-	return ctx.OK(&response)
+	return ctx.ConditionalEntities(resultUsers, c.config.GetCacheControlCollaborators, func() error {
+		data := make([]*app.UserData, len(s))
+		for i := range resultUsers {
```

Comment:
well, actually, I need to keep the array of `account.User` because that's what the `ctx.ConditionalEntities()` method requires as the first parameter. Moving the conversion from `account.User` to `app.User` is the pattern that we've used on other endpoints, because there's no need to perform the conversion if the actual data did not change since the client's last request (performing comparison with the `If-None-Match` or `If-Modified-Since`  request headers).